src/tests/ftest/pool/create_capacity.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
```

Comment:
If you need to push again - can be removed

Reply:
Done